browser/css/cool.css

```diff
@@ -742,10 +763,10 @@ body {
 	box-sizing: content-box !important;
 	max-width: 32px;
 	display: inline-block;
-	border: solid 2px;
+	border: solid 1px;
 	border-radius: 50%;
-	height: 32px;
-	width: 32px;
+	height: var(--default-height);
+	width: var(--default-width);
 	padding: 0;
 	opacity: 0.9;
 }
```

Comment:
There is a problem with the opacity, it never gets to fully opaque (1), can you please check that? Historically,  I think it was set to that opacity only if the card is not active (?) can you recheck git blames etc?